src/status_im/protocol/listeners.cljs

```diff
@@ -24,6 +25,11 @@
     (decrypt key content)
     content))
 
+(defn normalize-identity [identity]
+  (if (and identity (s/starts-with? identity "0x"))
+    (subs identity 2)
+    identity))
```

Comment:
hm
there are also duplicates of this function in `status-im.models.pending-messages` (line 22) and `status-im.transactions.subs` (line 22). maybe we should use this function everywhere instead? (it is not related to this task, but anyway)